agasc/agasc.py

```diff
@@ -177,6 +191,13 @@ def get_agasc_cone(ra, dec, radius=1.5, date=None, agasc_file=None,
     corrected positions are available in the ``RA_PMCORR`` and ``DEC_PMCORR``
     columns, respectively.
 
+    If ``use_mag_est`` is ``True``, then stars with available mag estimates in
+    the AGASC supplement are updated in-place in the output ``stars`` Table:
+
+    - ``MAG_ACA`` and ``MAG_ACA_ERR`` are set according to the supplement.
+    - ``MAG_CATID`` (mag catalog ID) is set to ``MAG_CATID_SUPPLEMENT`` (128).
+    - If COLOR1 is 0.7 or 1.5 then it is changed to 0.69 or 1.49 respectively.
```

Comment:
Not sure if we might want a clarifying comment indicating "if the COLOR1 hasn't already been updated based on APASS" or some such.